<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>361</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>358</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(22, 26, 35);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignVCenter | AlignRight';
	color: rgb(158, 163, 182);
}



</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAC">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>63</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PlusMin">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>63</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_slesh">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>63</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>63</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>181</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>181</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>181</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>181</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>299</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>299</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 58, 78);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #5f688c , stop:1 #7981a3);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>239</y>
      <width>61</width>
      <height>122</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(83, 179, 105);
	color: white;
	border: 2px solid black;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignVCenter | AlignRight';
	color: rgb(158, 163, 182);
}



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sec">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>299</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arctg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>299</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>arctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>181</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arcsin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>181</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>arcsin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arccos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>arccos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_grad">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>63</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>degree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backspace">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>63</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(40, 46, 60);
	color: rgb(158, 163, 182);
	border: 2px solid black;
}

QPushButton::pressed{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #3e475c , stop:1 #677699);
}</string>
    </property>
    <property name="text">
     <string>BSpace</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
